blog/migrate-to-webcontentsview.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Migrating from BrowserView to WebContentsView
+date: 2024-11-11T00:00:00.000Z
+authors: yangannyx
+slug: migrate-to-webcontentsview
```

Comment:
is it alright that the slug doesn't exactly match the title?

Reply:
There is precedent for this in a few of our blog files, [for a recent example](https://github.com/electron/website/blob/main/blog/rfcs.md?plain=1), so I would say it is fine.